src/HtmlGenerator/Pass1-Generation/GenerateFromBuildLog.cs

```diff
@@ -16,34 +16,6 @@ public class GenerateFromBuildLog
         public static readonly Dictionary<string, string> AssemblyNameToFilePathMap =
             new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
-        public static void IndexLogFile(string logFile, string serverPath = null, string solutionRoot = null)

```

Comment:
Yes, this is dead and can go. This was parsing MSBuild text log files before binlogs were around.